src/window/castlewindow.pas

```diff
@@ -957,6 +957,14 @@   TCastleWindow = class(TComponent)
     procedure UpdateRequirements;
   protected
     procedure DoUpdate; virtual;
+    { procedures for dropping files }
```

Comment:
Move these lines into `src/window/windows/castlewindow_winapi.inc`, place them between

```
{$ifdef read_window_interface}
...
{$endif read_application_interface}
````

This will achieve the same effect (the methods will be part of `TCastleWindow` class), but will keep nicely everything in `src/window/windows/castlewindow_winapi.inc`. So you will not need to surround it with `{$ifdef CASTLE_WINDOW_WINAPI}...{$endif}`.

Reply:
Okay, i will try that. Not having the $endif match the $ifdef is confusing

Reply:
I made the

{$ifdef read_window_interface}

match the $endif.  I saw one other place that had this.
{$endif read_window_interface}

Reply:
I don't see any problem with replacing read_application_interface with read_window_interface. Resolved, I hope.